install_config/cluster_metrics.adoc

```diff
@@ -289,6 +289,12 @@ ifdef::openshift-origin[]
 /usr/share/openshift/examples/infrastructure-templates/origin/metrics-deployer.yaml
 ----
 ====
+
+[NOTE]
```

Comment:
Added.

But, is this a thing we've recently started doing? It's not needed, and I wasn't aware we were doing it until the last time you asked me to add them...